DTS-STN/Life-Journeys
# Fixed axe accessibility issues + other small changes

# Description

[LJ-115 Fix small issues identified by axe scan](https://lj-decd.atlassian.net/browse/LJ-115?atlOrigin=eyJpIjoiNTMxYzU0N2M5Y2JkNDZhODkxZmQwNjBkYWEwNTJkMWEiLCJwIjoiaiJ9)

Fixed the issues identified in the Jira task, removed the Main component as it was unnecessary. Banner is now implemented by including the tag on the page you want a banner (attributes are no longer specified in Layout). Breadcrumbs were moved out of Layout and on to each page since keeping it above ```{children}``` in the main tag in Layout would put them above the banner on our second page.

## Test Instructions

1. Pull in branch
2. Type ```npm run dev```
3. Press F12 to open Dev Tools and run axe DevTools. The only issues remaining should be color contrast issues and "Links with the same name have a similar purpose" (which is referring to the "Explore this Journey" links).

## Meets Definition of Done
